setup.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from setuptools import setup
+
+packages = \
+['exasol_transformers_extension']
+
+package_data = \
+{'': ['*']}
+
+install_requires = \
+['Jinja2>=3.0.3,<4.0.0',
+ 'exasol-bucketfs-utils-python @ '
+ 'git+https://github.com/exasol/bucketfs-utils-python.git@main',
+ 'importlib-resources>=5.4.0,<6.0.0']
```

Comment:
Code duplication. Is it necessary to have both setup.py and pyproject.toml?

Reply:
this file is generated

Reply:
Please add a comment to the file that it is generated.

Reply:
we can add an explanation into the developer guide